From 2cbcf323e8e28d4a9af15b7bd1adac35dba788b6 Mon Sep 17 00:00:00 2001
From: Lu Feifei <lufeifei@wxiat.com>
Date: Fri, 15 Jul 2022 13:47:08 +0800
Subject: [PATCH] sw64: delete run_mode in struct cpu_desc_t
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 2cbcf323e8e28d4a9af15b7bd1adac35dba788b6
Modified-by-SEL: No


Sunway inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I56OLG

--------------------------------

Signed-off-by: Lu Feifei <lufeifei@wxiat.com>
Signed-off-by: Gu Zitao <guzitao@wxiat.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/sw_64/include/asm/hw_init.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/sw_64/include/asm/hw_init.h b/arch/sw_64/include/asm/hw_init.h
index f60a58570a92..71b569b6ec98 100644
--- a/arch/sw_64/include/asm/hw_init.h
+++ b/arch/sw_64/include/asm/hw_init.h
@@ -45,7 +45,6 @@ struct cpu_desc_t {
 	char vendor_id[16];
 	char model_id[64];
 	unsigned long frequency;
-	__u8 run_mode;
 } __randomize_layout;
 
 #define MAX_NUMSOCKETS		8
-- 
2.33.0

